### PR TITLE
Restyle button link component

### PR DESCRIPTION
**Pull request by vazquez**
Before:
screenshot 2015-08-10 16 10 32

After:

screenshot 2015-08-10 16 10 49

### CODE REVIEW

**Comment by xmatthewx** [comment]
Is getContrastingColor used for button label color? I think we've had some trouble with that.

File: src/lib/color.js [hunk 1]
```diff
@@ -13,5 +13,11 @@ module.exports = {
           bc = black.contrast(c);
       return (wc < bc && wc >= minimumContrast) ? whiteCSS : blackCSS;
     };
-  }())
+  }()),
+  darken: function(baseColor, shade){
+    var color = Color(baseColor);
+    color.darken(shade);
+
+    return color.hexString();
```
Comment (k88hudson) [inline src/lib/color.js]: can you use `rgbString` instead to account for transparent colors?

**Comment by k88hudson** [comment]
@xmatthewx yes I've noticed a problem with that too, it's unrelated to this patch though

**Comment by xmatthewx** [comment]
> it's unrelated to this patch though 

As long as it won't yield ugly buttons, ... cool.

**Comment by k88hudson** [comment]
R+! :clap:

**Pull request merged**